### DISCUSSION TITLE
v0.4.28

### DISCUSSION THREAD

**Discussion by lbbrhzn** [lbbrhzn/ocpp#460] [Announcements]
## Changes

* Add extra states to Charge Control switch (#459) @drc38

## :package: Dependencies

* build(deps): bump pytest-homeassistant-custom-component from 0.7.5 to 0.7.6 (#458) @dependabot

<hr /><em>This discussion was created from the release <a href='https://github.com/lbbrhzn/ocpp/releases/tag/v0.4.28'>v0.4.28</a>.</em>

**Comment by McFranky** [comment]
Hey folks

After installing the new version v0.2.28 I noticed, that after charging the charge control switch don't switch off. Returning to v.0.2.27 the switch reacts normal after charging (switch off)

v0.2.28
![grafik](https://user-images.githubusercontent.com/59682922/161239703-fa9a226d-8c9d-4a97-9661-87275a748b95.png)

v0.2.27
![grafik](https://user-images.githubusercontent.com/59682922/161239950-74ae2a1c-aff6-434a-afc7-ed1bb3336d09.png)



Thanks for notice
Frank

**Reply by drc38** [reply]
In the SuspendedEV and SuspendedEVSE states the switch will now be "On" from 0.4.28. This is because the charging transaction has not ended. If you turn it "Off" the charge transaction will stop and the status will go to Finishing. In the Finishing state the switch will be "Off".
